types/yeoman-generator/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for yeoman-generator 5.0
+// Type definitions for yeoman-generator 5.3.0
```

Comment:
Please remove the trailing `.0` as specifying patch-versions isn't allowed and thus cause the tests to fail.

You might want to write `5.2` here as this was (as far as I can see) the first version that introduced features in that manner.

Reply:
Done